files/image_config/updategraph/updategraph

```diff
@@ -106,12 +101,15 @@ if [ "$src" = "dhcp" ]; then
     fi
     if [ "`cat /tmp/dhcp_graph_url`" = "N/A" ]; then
         echo "'N/A' found in DHCP response. Skipping graph update and generating an empty configuration."
-        if [ ! -f /etc/sonic/minigraph.xml ]; then
-            copy_default_minigraph
+        PLATFORM=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
+        if [ -f /etc/sonic/minigraph.xml ]; then
+            sonic-cfggen -H -m /etc/sonic/minigraph.xml --preset empty > /tmp/device_meta.json
+        elif [ -f /usr/share/sonic/device/$PLATFORM/default_sku ]; then
+            PRESET=(`head -n 1 /usr/share/sonic/device/$PLATFORM/default_sku`)
+            sonic-cfggen -H -k ${PRESET[0]} --preset empty > /tmp/device_meta.json
+        else
```

Comment:
not sure we need this else. in my opinion, we should get rid of the default minigraph.xml completely from the image.

Reply:
If we remove this `else` block, we'll need to make sure all the new platforms have a `default_sku` file in there device folder. I can create them for all existing ones, but we'd better communicate with community on this change for the future ones.

Reply:
You may create a unit test in sonic-buildimage/src/sonic-device-data/ to enforce any rule you want and later it will be added into PR check.

Another option: in stead of fallback to minigraph, how about fallback to the SKU with shortest name?

Reply:
@taoyl-ms , agree with you about the announcement. why not talk about this next community meeting?